docs/io/unified.rst

```diff
@@ -309,7 +306,10 @@ round-tripping the object with no loss of precision.  For example::
     >>> tm['a'] == t['a']
     array([ True,  True], dtype=bool)
 
-The same will work with ``QTable``.
+Note that even with ``astropy_native=True``, numerical columns with units will
```

Comment:
Somehow this new text implies that `Column` is not an astropy native object.  I think that this whole paragraph is confusing in this spot because it is a hard segway from `Time` to a somewhat unrelated topic of `QTable` vs. `Table` handling of `Quantity`.  I liked the original "The same will work with QTable".

But if we really want this paragraph then change `stored` to `read`, because `stored` implies that the sentence is related to writing, not reading.  (I don't think of "storing" columns into a table in memory).